### PR TITLE
Add initial CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Kupoman**
Adding an initial CONTRIBUTING.md as recommended by the [community profile](https://github.com/panda3d/panda3d/community). The intent is to get this started now and expand on it later.

Pull request #777 also starts a CONTRIBUTING.md file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributing to Panda3D
+Thank you for looking into ways to contribute to Panda3D.
+Most information on how to contribute and get involved already exists in other places, so this page serves as another way to find those resources.
```
Comment (dstokesf5) [inline CONTRIBUTING.md:3]: The "so" in this line is really starting to bother me, I'm going to reword this.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* [Manual](https://docs.panda3d.org/1.10/python/index)
+
+### Getting in Touch With the Community
+* [Discord](https://discord.gg/UyepRMm)
+* [IRC](https://www.panda3d.org/manual/?title=The_IRC_Channel)
+* [Forum](https://discourse.panda3d.org/)
+
+
+## Building the Project
+Information on how to build Panda3D can be found in the [README](https://www.panda3d.org/get-involved/).
```
Comment (Moguri) [inline CONTRIBUTING.md:16]: I believe you wanted this to point to the readme and not the get involved page.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Testing the Project
+Information on how to test Panda3D can be found in the [README](https://www.panda3d.org/get-involved/).
```
Comment (Moguri) [inline CONTRIBUTING.md:19]: I believe you wanted this to point to the readme and not the get involved page.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Kupoman) [reply CONTRIBUTING.md]: Fixed.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Kupoman) [reply CONTRIBUTING.md]: Fixed.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Contributing to Panda3D
+Thank you for looking into ways to contribute to Panda3D.
+Most information on how to contribute and get involved already exists in other places,
```
Comment (Moguri) [inline CONTRIBUTING.md:3]: This sentence is punctuated with a comma instead of a period.

**Comment by Moguri** [comment]
The content looks good to me, and we can expand on it later as needed.

File: CONTRIBUTING.md (hunk 4 above)
Comment (Kupoman) [reply CONTRIBUTING.md:3]: Fixed.

**Comment by rdb** [comment]
Thanks for initiating this.  However, this is more of a loose collection of links to resources, some of which not related to contributing.  I've committed a file that goes into a bit more depth about the process of contributing.  Further suggestions for changes are always welcome of course, but it is a start.

**Pull request closed without merging**